tests/lib/rules/boolean-prop-naming.js

```diff
@@ -417,6 +417,56 @@ ruleTester.run('boolean-prop-naming', rule, {
       features: ['ts'],
       errors: [],
     },
+    {
+      code: `
+        type Props = {
+          isEnabled: boolean
+        } & OtherProps
+        const HelloNew = (props: Props) => { return <div /> };
+      `,
+      options: [{ rule: '^is[A-Z]([A-Za-z0-9]?)+' }],
+      features: ['ts'],
```

Comment:
does this pass with flow as well?
```suggestion
      features: ['types'],
```

same question throughout

Reply:
@ljharb yes, seems it does! 🎉